### ISSUE TITLE
Launch a terminal by double-click?

### ISSUE THREAD

**Issue by JimMeLad** [SchoofsKelvin/vscode-sshfs#428]
Hi,
Just installed your extension and have it working OK so far, but I was wondering if there was an easier way to launch a terminal session eg by double-clicking or right-clicking the configuration name rather than having to click the (rather small) 'Open remote SSH terminal' icon?
![image](https://github.com/user-attachments/assets/1a5a0614-2aed-4e31-b386-ea0984d859c8)

**Comment by luckman212** [comment]
Or, any way to configure the saves config to automatically open a Remote Terminal?
something like:
```json
{
    "agent": "$SSH_AUTH_SOCK",
    "host": "pve02",
    "name": "pve02",
    "port": 22,
    "username": "root",
    "autostart_terminal": true  <==
}
```

**Comment by SchoofsKelvin** [comment]
No, the extension API doesn't have a way to detect double-clicking.

As for automatically opening a terminal, I wouldn't add that to configs, as those are usually workspace-independent and would mean the terminal would open every time you open a new VS Code window. 
Alternatively, perhaps a `ssh.autoStartTerminals` "User Setting" (but meant to be used in your workspace/folder settings) could work. Feel free to create an issue for that if that's something you'd like.
